apps/src/code-studio/components/progress/UnitCalendar.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import UnitCalendarLessonChunk from './UnitCalendarLessonChunk';
+import {unitCalendarLessonChunk} from '@cdo/apps/templates/progress/unitCalendarLessonShapes';
+
+const weekWidth = 592;
+const styles = {
+  weekColumn: {
+    minWidth: 100,
+    backgroundColor: 'rgb(118, 101, 160)',
+    color: 'white',
+    textAlign: 'center',
+    border: '1px solid rgb(118, 101, 160)',
+    borderCollapse: 'collapse',
+    fontWeight: 'bold',
+    minHeight: 50
+  },
+  scheduleColumn: {
+    border: '1px solid rgb(118, 101, 160)',
+    borderCollapse: 'collapse',
+    width: weekWidth,
+    display: 'flex',
+    minHeight: 50
+  },
+  table: {
+    border: '1px solid rgb(118, 101, 160)',
+    borderCollapse: 'collapse',
+    width: '100%'
+  },
+  key: {
+    border: '1px solid rgb(118, 101, 160)',
+    borderCollapse: 'collapse',
+    width: '100%',
+    marginTop: 20
+  },
+  keyIcon: {
+    marginRight: 5
+  },
+  keySection: {
+    display: 'flex',
+    justifyContent: 'space-around',
+    width: '90%',
+    alignItems: 'center',
+    fontSize: 15
+  }
+};
+
+export default class UnitCalendar extends React.Component {
+  static propTypes = {
+    schedule: PropTypes.arrayOf(PropTypes.arrayOf(unitCalendarLessonChunk))
+      .isRequired,
+    weeklyInstructionalMinutes: PropTypes.number
+  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      hovering: ''
+    };
+  }
+  handleHover = id => {
```

Comment:
Once you start using Radium I don't think you will need the hovering state work you have in this file.